.circleci/config.yml

```diff
@@ -3,37 +3,25 @@ version: 2.1
 jobs:
   test-linux: &test-linux-template
     parameters:
-      dimod-version:
-        type: string
       python-version:
         type: string
       integration-test-python-version:
         type: string
 
     docker:
-      - image: circleci/python:<< parameters.python-version >>
+      - image: cimg/python:<< parameters.python-version >>
 
     steps:
       - checkout
 
-      - restore_cache: &restore-test-cache-template
```

Comment:
Why drop caching?

Reply:
Turns our loading the cache in circleci is often slower than just a fresh download